modules/secrets_manager/locals.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   # Description of the KMS key
-  description = "KMS key for ${local.namespace}-service"
+  description = "KMS key for CloudWatch service"
```

Comment:
I think KMS will be used for app secrets as well, so it is not only for cloudwatch

Reply:
I rolled it back in fb3307b ✅